stake-pool/program/tests/set_funding_authority.rs

```diff
@@ -17,7 +17,7 @@ use {
     },
```

Comment:
tangential discussion but since when was github able to do this with recognizing file name changes? :heart_eyes:

Reply:
Huh, that's a good point, i'm not sure, but it's *very* welcome :smile: